src/rdb.c

```diff
@@ -2712,7 +2712,7 @@ void rdbLoadProgressCallback(rio *r, const void *buf, size_t len) {
     }
 }
 
-static int rdbFunctionLoad(rio *rdb, int ver, functionsCtx* functions_ctx) {
+static int rdbFunctionLoad(rio *rdb, int ver, functionsCtx* functions_ctx, int flags) {
```

Comment:
we should probably rename the new arg to `rdbflags`, or document which "enum" it belongs to (or both)

Reply:
Change the name, do you want me to also document the function? Or is it clear enough?